src/config.rs

```diff
@@ -66,12 +66,18 @@ fn build_cli_parser<'a, 'b>() -> App<'a, 'b> {
                 .long("git-tag")
                 .help("Optional commit the updated version and create a git tag."),
         )
+        .arg(
+            Arg::with_name("ignore-lockfile")
+                .long("ignore-lockfile")
+                .help("Don't update the lockfile")
```

Comment:
following the other suggestion and to not introduce a breaking change, let's change this flag to:

```suggestion
            Arg::with_name("build")
                .long("build")
                .help("Run cargo build after version change to generate Cargo.lock")
```